hbw/config/config_2017.py

```diff
@@ -472,6 +472,8 @@ def make_jme_filename(jme_aux, sample_type, name, era=None):
         "Muon.pt", "Muon.eta", "Muon.phi", "Muon.mass",
         "Electron.pt", "Electron.eta", "Electron.phi", "Electron.mass",
         "MET.pt", "MET.phi",
+        # Gen properties
+        "gen_hbw_decay.*",
```

Comment:
I think we can remove these from here, we don't want to save genparticles after ReduceEvents, right?